addons/web/static/tests/views/list_view_tests.js

```diff
@@ -15138,26 +15138,35 @@ QUnit.module("Views", (hooks) => {
             patchWithCleanup(browser.localStorage, {
                 getItem(key) {
                     assert.step("getItem " + key);
-                    return forceLocalStorage ? "m2o" : this._super(arguments);
```

Comment:
this test is already really long. We encourage small, well scoped, tests. In your case, it would be nice to have a test "list view with optional fields from local storage being the empty array"